deepmd/pt/model/atomic_model/base_atomic_model.py

```diff
@@ -455,13 +455,22 @@ def change_out_bias(
             )
             self._store_out_stat(delta_bias, out_std, add=True)
         elif bias_adjust_mode == "set-by-statistic":
+            intensive = (
+                self.fitting_net.intensive
+                if (
+                    hasattr(self, "fitting_net")
+                    and hasattr(self.fitting_net, "intensive")
+                )
```

Comment:
not good idea to access data member of class object.

Reply:
I have implemented `get_intensive` method in `atomic_model` layer. And change this sentence to `intensive = self.get_intensive()`. Is it elegant?